### ISSUE TITLE
Custom shortcuts don't work after locking and unlocking screen

### ISSUE THREAD

**Issue by lmichelin** [hwtnb/SylphyHornPlusWin11#21]
Hello, I configured the following shortcuts:

![image](https://user-images.githubusercontent.com/33673240/207848421-04310024-9b84-4a65-9f38-ea2adb8e2c39.png)

But when I lock the screen with WIN+L then unlock it, these shortcut don't work anymore, I need to quit the program and relaunch it.

App version: 4.0.0 beta.9

OS: Windows 11 Pro
Version: 22H2
OS build: 22621.963
Experience: Windows Feature Experience Pack 1000.22638.1000.0

I hope you will be able to reproduce the issue.

Thanks for your support.

**Comment by johnrs** [comment]
I'm a new user.  I am seeing the same, or very similar.  After I lock, then unlock, sometimes (most of the time) my desktop shortcut keys aren't working.  Then I noticed that "ctrl <-" and "ctrl ->" were doing what normally "ctrl win <-" and "ctrl win ->" do. by default.   I only noticed this because my text editor uses the "ctrl <-" and "ctrl ->" sequences.

Furthermore, if I use the "ctrl win <-" and "ctrl win ->" sequences, then the "ctrl <-" and "ctrl ->" sequences go back to not doing anything, AND my chosen desktop short keys work.  (later)  Indeed, it seems that if I use the win key for anything, then it all goes back to normal.

In other words - It seems that upon unlocking,sometimes the win key seems to be stuck on, till you use it.

I'm running Windows 10, 21H2, 19044.2251.

I hope this helps.  Thanks.
